cleanup/cleanup_test.go

```diff
@@ -0,0 +1,302 @@
+						},
+					},
+				},
+			},
+		}
+
+		serverSucceeded = &corev1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "server-succeeded",
+				Labels: map[string]string{
+					config.LoadTestLabel:      "LoadTest",
+					config.RoleLabel:          config.ServerRole,
+					config.ComponentNameLabel: "name",
+				},
+			},
+			Status: corev1.PodStatus{
+				InitContainerStatuses: []corev1.ContainerStatus{
+					{
+						State: corev1.ContainerState{
+							Terminated: &corev1.ContainerStateTerminated{ExitCode: 0},
+						},
+					},
+					{
+						State: corev1.ContainerState{
+							Terminated: &corev1.ContainerStateTerminated{ExitCode: 0},
+						},
+					},
+				},
+				ContainerStatuses: []corev1.ContainerStatus{
+					{
+						State: corev1.ContainerState{
+							Terminated: &corev1.ContainerStateTerminated{ExitCode: 0},
+						},
+					},
+				},
+			},
+		}
+
+		// This is one of the possibility that the pod would be marked as errored,
+		// testing other possibilities is out of scope of this test and done
+		// elsewhere.
+		clientErrored = &corev1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "client-errored",
+				Labels: map[string]string{
+					config.LoadTestLabel:      "LoadTest",
+					config.RoleLabel:          config.ClientRole,
+					config.ComponentNameLabel: "name",
+				},
+			},
+			Status: corev1.PodStatus{
+				InitContainerStatuses: []corev1.ContainerStatus{
+					{
+						State: corev1.ContainerState{
+							Terminated: &corev1.ContainerStateTerminated{ExitCode: 0},
+						},
+					},
+					{
+						State: corev1.ContainerState{
+							Terminated: &corev1.ContainerStateTerminated{ExitCode: 0},
+						},
+					},
+				},
+				ContainerStatuses: []corev1.ContainerStatus{
+					{
+						State: corev1.ContainerState{
+							Terminated: &corev1.ContainerStateTerminated{ExitCode: 127},
+						},
+					},
+				},
+			},
+		}
+
+		serverErrored = &corev1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "server-errored",
+				Labels: map[string]string{
+					config.LoadTestLabel:      "LoadTest",
+					config.RoleLabel:          config.ServerRole,
+					config.ComponentNameLabel: "name",
+				},
+			},
+			Status: corev1.PodStatus{
+				InitContainerStatuses: []corev1.ContainerStatus{
+					{
+						State: corev1.ContainerState{
+							Terminated: &corev1.ContainerStateTerminated{ExitCode: 0},
+						},
+					},
+					{
+						State: corev1.ContainerState{
+							Terminated: &corev1.ContainerStateTerminated{ExitCode: 0},
+						},
+					},
+				},
+				ContainerStatuses: []corev1.ContainerStatus{
+					{
+						State: corev1.ContainerState{
+							Terminated: &corev1.ContainerStateTerminated{ExitCode: 127},
+						},
+					},
+				},
+			},
+		}
+
+	})
+
+	It("doesn't send quit to driver", func() {
+		podList = append(podList, driver)
+
+		quitWorkers(ctx, mockQuit, podList, log)
+		Expect(mockQuit.called).To(BeEmpty())
+	})
+
+	It("doesn't send quit to Succeeded workers", func() {
```

Comment:
nit: let's use the same case as in line 278 and line 285.

```suggestion
	It("doesn't send quit to succeeded workers", func() {
```